Dockerfile

```diff
@@ -1,34 +1,22 @@
-FROM openjdk:8u212-jdk-stretch AS builder
+FROM openjdk:8u212-jre-slim-stretch
 
-RUN mkdir -p /opt/cassy
-WORKDIR /opt/cassy
+WORKDIR /cassy
 
-COPY gradle ./gradle
-COPY build.gradle ./
-COPY gradlew ./gradlew
-COPY src ./src
+# The Docker context is created in build/docker by `./gradlew docker`
+COPY ./cassy.tar .
+RUN tar xf cassy.tar -C / && rm -f cassy.tar
 
-RUN ./gradlew installDist
-
-# Runtime Container
-FROM openjdk:8-alpine
-
-RUN apk add sqlite
-
-WORKDIR /opt/cassy
-
-RUN mkdir -p /cassy/data && mkdir -p /cassy/conf
-
-# Copy Build output from Builder Step
-COPY --from=builder /opt/cassy/build ./build
 COPY scripts ./scripts
-COPY conf ./conf
+COPY logback.xml entrypoint.sh ./
 
-RUN sqlite3 ./cassy.db < ./scripts/db.schema
+RUN apt-get update && apt-get install -y \
+    sqlite \
+    && rm -rf /var/lib/apt/lists/*
 
-VOLUME /cassy
+RUN mkdir -p /cassy/data /cassy/conf
 
-ENTRYPOINT ["/bin/sh", "scripts/entrypoint.sh"]
-CMD ["--config", "/cassy/conf/cassy.properties"]
+ENV JAVA_OPTS -Dlogback.configurationFile=/cassy/logback.xml
+ENTRYPOINT ["/cassy/entrypoint.sh"]
```

Comment:
Just a question. Any reason of using `ENTRYPOINT` here instead of just `RUN` ?
Also, It looks a bit weird since it is not an entrypoint.

Reply:
To persist the database file, it should be created outside the container. So the database must be initialized after the container starts. If we do it with `RUN`, the database is only created inside the image and it is not persisted.
It's a common way to Initialize a database with entrypoint. https://success.docker.com/article/use-a-script-to-initialize-stateful-container-data

Reply:
Oh, I didn't know that. Thanks !